packages/manager/apps/container/src/container/legacy/account-sidebar/UsefulLinks/Link/Anchor.tsx

```diff
@@ -30,8 +30,8 @@ const Anchor = ({
   return (
     <a
       className="d-flex"
-      target="_blank"
-      rel="noreferrer"
+      target={link.external ? '_blank' : ''}
+      rel={link.external ? 'noreferrer' : ''}
```

Comment:
```suggestion
      target={link.external ? '_blank' : '_self'}
      rel={link.external ? 'noopener' : 'opener'}
```
I don't know if is best practice to have the empty html tag. Maybe can be filled with the opposite value.